scripts/variantstore/wdl/GvsCallsetCost.wdl

```diff
@@ -2,11 +2,11 @@ version 1.0
 
 workflow GvsCallsetCost {
     input {
-#        String project_id
-#        String dataset_name
+        String project_name
+        String dataset_name
         String workspace_namespace
         String workspace_name
-#        String callset_name
+        String callset_name
```

Comment:
what is the callset name? is this the same as the filter_set_name ?

Reply:
Actually this should be callset *identifier*, which is the name of a column in the cost observability table. It looks like the plan is to [default the filter set name to the call set identifier](https://github.com/broadinstitute/gatk/pull/7915/files#diff-e03ceae55b79cd7b7eb8f85cf754f3d8db9b635e00283d947b6276d0b7d9eb62R37) unless a value is explicitly specified.